controller/setting_controller.go

```diff
@@ -859,7 +859,7 @@ func (bst *BackupStoreTimer) Start() {
 			return false, err
 		}
 
-		log.Debug("Triggering sync backup target")
+		log.Info("Triggering sync backup target")
```

Comment:
Can be debug to prevent repeated message. If having any errors, the error will be logged, and we already have `staring` and `stopped backup store timer`, so a repeated info probably is not really required. WDYT?

Reply:
Updated. Thank you.